app/services/search/user.rb

```diff
@@ -10,7 +10,12 @@ class << self
       def search_usernames(username)
         results = search(body: username_query(username))
         results.dig("hits", "hits").map do |doc|
-          doc.dig("_source", "username")
+          source = doc["_source"]
+          {
+            "username" => source["username"],
+            "name" => source["username"],
```

Comment:
**question:** Why is this `source["username"]` instead of `source["name"]`?

I noticed we do this already too:
https://github.com/forem/forem/blob/f975b8b73767e63f92c2d2579bd421b130e29cf9/app/services/search/user.rb#L34-L38

Reply:
Haha bc that is where I copied it from and likely that's needed on the frontend for some special things, here though it should be name, good catch!